lib/game_teams_manager.rb

```diff
@@ -99,15 +105,10 @@ def game_teams_by_season(season)
   end
 
   def team_tackles(season)
-    team_season_tackles = {}
-    game_teams_by_season(season).each do |game|
-      if team_season_tackles[game.team_id]
-        team_season_tackles[game.team_id] += game.tackles
-      else
-        team_season_tackles[game.team_id] = game.tackles
-      end
+    game_teams_by_season(season).reduce(Hash.new(0)) do |team_season_tackles, game|
+      team_season_tackles[game.team_id] += game.tackles
+      team_season_tackles
```

Comment:
This looks much better. Thanks for cleaning it up!

Reply:
Happy to do so!